integrations/standalone/tests/integration/parts/task.ts

```diff
@@ -17,28 +14,32 @@ export class TasksTester implements PartTest {
   partName() {
     return 'Tasks';
   }
-  async fill(page: Page) {
+  async fill(part: Part) {
     for (const task of this.tasks) {
-      await TabUtil.change(page, task.tab);
-      await task.test.fill(page);
+      const tab = (part as Accordion).tab(task.tab);
+      await tab.switchTo();
```

Comment:
maybe not possible (I didn't see all the code yet), but I would expect a general tab object and the switchTo method should get the name of the tab. So the reading is the tabs should switch to the tab name...